.github/ISSUE_TEMPLATE/bug-issue.yml

```diff
@@ -48,19 +48,27 @@ body:
 
         Attaching log files is required for every reported bug. See instructions below on how to find them.
 
+        **Logs are reset when you reopen the game.** If the game crashed or has been closed since you found the bug, retrieve the logs using the file explorer instead.
+
+        ### Desktop platforms
+
         If the game has not yet been closed since you found the bug:
           1. Head on to game settings and click on "Open osu! folder"
           2. Then open the `logs` folder located there
 
-        **Logs are reset when you reopen the game.** If the game crashed or has been closed since you found the bug, retrieve the logs using the file explorer instead.
-
-        The default places to find the logs are as follows:
+        The default places to find the logs on desktop platforms are as follows:
           - `%AppData%/osu/logs` *on Windows*
           - `~/.local/share/osu/logs` *on Linux & macOS*
+
+        If you have selected a custom location for the game files, you can find the `logs` folder there.
+
+        ### Mobile platforms
```

Comment:
Not sure I care for any of these mobile changes. It's just creating more text for people to read which makes it less likely that it will be read in full. I'd just preface the "Open osu! folder" section with a

```diff
- If the game has not yet been closed since you found the bug:
+ On desktop platforms, if the game has not yet been closed since you found the bug:
```

and call it a day IMO.

Reply:
I personally find adding headers to separate between platforms better than including more conditions in regular text, but will wait for other opinions.